apiary.apib

```diff
@@ -74,11 +74,14 @@ A Question object has the following attributes.
 
 This action allows you to vote on a question's choice.
 
-+ Response 201
++ Response 201 (application/json)
 
-    + Headers
+        {
+            "url": "/questions/1/choices/1",
+            "votes": 1,
+            "choice": "Swift"
+        }
 
-            Location: /questions/1
```

Comment:
I've changed the behaviour of the API in 3f902c30390ea476e34f386e232bfb7e039f5619 so it doesn't redirect a Location header to another resource.